src/main/java/com/uid2/shared/auth/Keyset.java

```diff
@@ -79,7 +78,7 @@ public boolean equals(Object o) {
                 && this.isEnabled == b.isEnabled
                 && this.isDefault == b.isDefault;
 
-        if(this.allowedSites == null || this.allowedSites == null) {
+        if(this.allowedSites == null || b.allowedSites == null) {
```

Comment:
A bit scary that none of us saw the original bug. You can use intellij to generate these methods.  Maybe we should have test coverage of this method?

Reply:
It was hard to found.

Reply:
how did you find it?